lib/AST/ASTImporter.cpp

```diff
@@ -141,6 +141,7 @@ namespace clang {
                            bool Complain = true);
     bool IsStructuralMatch(EnumDecl *FromEnum, EnumDecl *ToRecord,
                            bool Complain = true);
+    bool IsStructuralMatch(FunctionDecl *From, FunctionDecl *To);
```

Comment:
I wonder if it is a good pattern to have that many functions that do the same thing. In the future maybe this would be a great candidate to be refactored (i.e.: one templated implementation).